sources/kz/countrywide.json

```diff
@@ -0,0 +1,23 @@
+{
+    "coverage": {
+        "country": "kz",
+        "ISO 3166": {
+          "alpha2": "KZ", 
+          "country": "Kazakhstan"}
+    },
+    "data": "https://s3.amazonaws.com/data.openaddresses.io/cache/uploads/sergiyprotsiv/5f6b95/kz.csv.zip",
+    "attribution": "Комитет по правовой статистике и специальным учетам Генеральной прокуратуры Республики Казахстан",
+    "type": "http",
+    "compression": "zip",
+    "language": "ru",
+    "conform": {
+        "type": "csv",
+        "number": "number",
+        "street": "street",
+        "district": "district",
+        "region": "region",
+        "srs": "EPSG:32642",
```

Comment:
Huh, I'm surprised this worked. `esri2geojson` should have dumped to `EPSG:4326`.